src/Inkluzitron/appsettings.json

```
@@ -61,7 +61,8 @@
         "RollSucceeded": "{0} má štěstí proti {3}, padlo {6}/{7} (bylo potřeba {8}+).\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*",
         "RollFailed": "{0} má proti {3} smůlu, padlo {6}/{7} (bylo potřeba {8}+).\n*{1}% submissive, {2}% dominant vs. {4}% submissive, {5}% dominant*\n**Tímto neúspěšným pokusem {10} {0} o {9} bodů, které {11} {3}.**",
         "RollFailedLossGendered": [ "přišel/a", "přišel", "přišla" ],
-        "RollFailedGainGendered": [ "dostal/a", "dostal", "dostala" ]
+        "RollFailedGainGendered": [ "dostal/a", "dostal", "dostala" ],
+        "NegativePoints": "{0} nemůžeš použít tento příkaz, pokud máš negativni počet bodů."
```

```suggestion
        "NegativePoints": "Nemůžeš použít tento příkaz, protože máš negativní počet bodů."
```

Tam moc ten username nedává smysl

---

Asi tam byt nemusi no. V tom pripade se ale musi upravit i ty dalsi shity.

---

Huh? Tohle je jediný řetězec pro negativní body ne?

---

Je, ale pak v jinem souboru se to pouzivalo jako `string.Format(...)` kde se daval i dalsi parametr, ale uz jsem hodil commit, kde jsem to zmenil na vraceni samotneho stringu.